runNodeTests.cmd

```diff
@@ -1 +0,0 @@
-msbuild Webjobs.Script.proj /t:MochaTest
```

Comment:
We don't have any node unit tests anymore, because we no longer have an Edge.js wrapper?

Reply:
The node unit tests are moving to the node repo (https://github.com/Azure/azure-functions-nodejs-worker/tree/dev/test)